Clinical-Genomics/cgweb
# Upgrade nuxt dependencies

This PR fixes the problem with the security flawed dependencies of cgweb

How to test:
1. install on cgweb-stage on clinical-db
1. login in OP

Expected outcome: 
1. Install succeeds
1. login succeeds
